docs/01-start-here/01-installation-steps.md

```diff
@@ -61,8 +61,7 @@ You need a Mac or Linux PC (Ubuntu).
 
 1. All being well, you should be able to [run the app](#running-the-app)
 
-Manual
```

Comment:
I've modified this because having two equivalent section headers written in different ways was stressful